packages/wordle/app/auth.tsx

```
@@ -0,0 +1,364 @@
+								class="mx-auto flex h-full w-full max-w-md flex-col items-center justify-center space-y-6 px-8"
+								action={formAction}
+							>
+								<div class="w-full">
+									<label htmlFor="email" class="block text-sm font-medium text-gray-700">
+										Email address
+									</label>
+									<div class="mt-1">
+										<input
+											id="email"
+											required
+											autoFocus={true}
```

Biome (2.1.2)

[error] Avoid the autoFocus attribute.

Unsafe fix: Remove the autoFocus attribute.


(lint/a11y/noAutofocus)